docs/Developers/Review.rst

```diff
@@ -140,6 +140,11 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check if the changes in the PR have been pushed from the master
```

Comment:
I think it is a good idea to state "haven't been pushed to master" and then fix the later part of the sentence.